src/audio/copier/copier.c

```diff
@@ -630,10 +653,13 @@ static int create_ipcgtw(struct comp_dev *parent_dev, struct copier_data *cd,
 		goto e_buf;
```

Comment:
is the line above `drv->ops.free(dev);` still correct? You need a new error handling label to free `ipcgtw_data`

Reply:
Why not use ipcgtw_zephyr_free() here as well instead of calling the dev free op?

Reply:
I changed in later patch, seems too late, let me change in this PR.

Reply:
doesn't it have to jump to `e_dev` here? But as I said, I think a new label would be better to also put `ipcgtw_zephyr_free(ipcgtw_data);` there

Reply:
I made this mistake, I saw ipcgtw_free have rfree(dev), then think jump to e_buf is ok, you are right, should jump to e_dev.

Regarding add a  new labal, I admit add new label will make code looks more reasonable